pkg/setup/setup.go

```diff
@@ -85,7 +85,16 @@ func (s *Setup) Configure(flags *Flags, clusterURL string, attach bool) (*config
 	httpOpts := []httpclient.Option{
 		httpclient.Timeout(5 * time.Second),
 		httpclient.Logger(s.logger),
-		httpclient.NoFollow(),
+	}
+
+	// Make sure we follow redirects.
+	canonicalClusterURL, err := s.detectCanonicalClusterURL(cluster.URL(), httpOpts)
+	if err != nil {
+		return nil, err
+	}
+	if !strings.Contains(canonicalClusterURL, cluster.URL()) {
```

Comment:
Why using contains here and not a `!=`?

Reply:
Because I return `resp.Request.URL.String()` which will return the domain with an `/` at the end which often is not included by user-input. the `.Contains` function will cover both cases.